### ISSUE TITLE
Assuming you want to help with development, what's stopping you?

### ISSUE THREAD

**Issue by MikeNeilson** [opendcs/opendcs#362]
Honest question, what can be done to make things better, easier, understandable so you can start if you so choose.

**Comment by ktarbet** [comment]
If some issues were tagged 'good first issue' that may help.

**Comment by MikeNeilson** [comment]
> If some issues were tagged 'good first issue' that may help.

Totally, but personally I've never been a good judge of that. Do you think you'd be able to go through one or two and make a guess? I think some of the GUI ones, like #368 that I just opened, actually are good first issues. No major changes to the bulk of the code but gets some familiarity with some of the concepts.

**Comment by ktarbet** [comment]
Yes, I'm looking through some the issues.  I tagged one good-first-issue so far, and closed another static issue.
